### PR TITLE
Resolves #224: CloudTenant.upgradeTenantContract() use wrapper class …

### LINKED ISSUE #224: CloudTenant upgradeTenantContract method accepts JSON vs. wrapper class for argument

### PR DESCRIPTION
**Pull request by bsnyder788**
…for arg.

### CODE REVIEW

**Comment by cfsnyder** [comment]
conflicts on this one now @bsnyder788 . You just need to remove the index.ts files

**Comment by bsnyder788** [comment]
@cfsnyder fixed conflicts

File: src/sdk/iland.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
+import { AuthProvider } from './auth/';
```
[the remark this answers is not in the document]
Comment (bsnyder788) [reply src/sdk/iland.ts]: fixed

File: src/sdk/model/edge/edge.ts [hunk 2]
```diff
@@ -388,11 +388,12 @@ export class Edge extends Entity {
    * @param {StaticRouting} request
    * @returns {Promise<Task>}
    */
+
```
[the remark this answers is not in the document]
Comment (bsnyder788) [reply src/sdk/model/edge/edge.ts]: IDE added them when I fixed the lint errors on master.  Removed

**Pull request merged**